packages/nectar/src/sagas/errors/handleErrors/handleErrors.saga.test.ts

```diff
@@ -43,15 +40,16 @@ describe('handle errors', () => {
     userCertificate: ''
   }
 
-  test('receiving registrar server error results in retrying registration', async () => {
+  test('receiving registrar server error results in retrying registration and not putting error in store', async () => {
```

Comment:
Shouldn't we also have similar test for ErrorCodes.SERVER_ERROR and ErrorCodes.SERVICE_UNAVAILABLE (using e.g test.each)?

Reply:
Surely we should have more tests, but add them after below task is finished.

https://github.com/ZbayApp/monorepo/issues/445